modules/rules_cuda/0.1.2/presubmit.yml

```diff
@@ -0,0 +1,4 @@
+tasks:
+  verify_targets_linux:
+    name: Verify build targets
+    platform: ubuntu2004 
```

Comment:
An idea for better test coverage: If you set up a test module with a mock toolchain, you could at least do a build with `--nobuild`. Most Bzlmod-related failures appear in the analysis phase anyway.

Reply:
We do already test that bzlmod is working ok in the rules_cuda CI so I'm not sure that doing this would give anymore coverage than is already tested? Plans are to get a hermetic toolchain setup in the future at which point we can add a proper test module.

Reply:
Since rules_cuda doesn't really offer a non-trivial Bzlmod interface i think this isn't needed yet. If it does, BCR tests allow for features such as automatically testing downstream compatibility, but that can come in the future.